frontend/micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/MDMSAddV2.js

```diff
@@ -28,12 +28,12 @@ const MDMSAdd = ({ defaultFormData, updatesToUISchema, screenType = "add", onVie
   const [noSchema, setNoSchema] = useState(false);
   const [loadDependent, setLoadDependent] = useState([]);
   const [showErrorToast, setShowErrorToast] = useState(false);
-
+  
   const [showToast, setShowToast] = useState(false);
   const { moduleName, masterName } = Digit.Hooks.useQueryParams();
-
+  
   useEffect(() => {
-    setSession({ ...session, ...defaultFormData });
```

Comment:
why session was removed ? @nipunarora-eGov

Reply:
Initially updating with the default values(data from server). This session  state had data of some previous master and was populating incorrectly. @jagankumar-egov

Reply:
check again reverting that now?

Reply:
Updated @jagankumar-egov